make/common/modules/LauncherCommon.gmk

```diff
@@ -43,6 +43,14 @@ LAUNCHER_CFLAGS += -I$(TOPDIR)/src/java.base/share/native/launcher \
 MACOSX_PLIST_DIR := $(TOPDIR)/src/java.base/macosx/native/launcher
 JAVA_MANIFEST := $(TOPDIR)/src/java.base/windows/native/launcher/java.manifest
 
+ifeq ($(call isTargetOs, bsd), true)
```

Comment:
This is not how this works anymore. First of all, do you really need a statically built libjli? Currently, this is only used on AIX because of limitations in that OS to express relative paths in dynamic library resolution.

Reply:
Tbh, I really don't know. This is how the out-of-tree BSD port has it, so it's as is, not necessarily how it should be :)